src/components/form/InputSecureTextIcon.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import { TouchableOpacity } from 'react-native';
 
-import { colors, Icon } from '../../config';
+import { colors, consts, Icon, texts } from '../../config';
+
+const a11yText = consts.a11yLabel;
```

Comment:
you missed it here

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/673dec6e31683527ba307138ccec0ebce57e64fd